src/Bridges/lazy_bridge_optimizer.jl

```diff
@@ -41,6 +41,9 @@ mutable struct LazyBridgeOptimizer{OT<:MOI.ModelLike} <: AbstractBridgeOptimizer
     objective_bridge_types::Vector{Any}
     objective_node::OrderedDict{Tuple{DataType}, ObjectiveNode}
     objective_types::Vector{Tuple{DataType}}
+    # Cache for (F, S) -> BridgeType. Avoids having to look up the node in the
```

Comment:
Isn't it the run-time method lookup for `concrete_bridge_type` which is slow?

Reply:
Ah. Yes. I guess this is what I meant.